fbw-a380x/src/wasm/systems/a380_systems/src/air_conditioning/mod.rs

```diff
@@ -266,6 +336,22 @@ impl A380AirConditioningSystem {
                     ElectricalBusType::AlternatingCurrent(2),
                 ],
             ),
+            fdac: [
+                FullDigitalAGUController::new(
+                    1,
+                    vec![
+                        ElectricalBusType::DirectCurrentEssential, // 403XP
+                        ElectricalBusType::AlternatingCurrent(1),  // 117XP
```

Comment:
```suggestion
                        ElectricalBusType::AlternatingCurrent(2),  // 117XP
```
117XP is powered by AC BUS 2

Reply:
Yeah... I messed up all the electrical buses 😥, thanks for noticing, fixed now to match all your suggestions.

Note: right now the FDAC is set up to run with one _**or**_ the other electrical bus powered. In reality each bus should power each of the channels. This will be improved and implemented in the next PR that I already have in work.